fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java

```diff
@@ -635,7 +634,7 @@ public List<List<String>> getInfos() {
                 List<String> cols = groupSchema.getDistributionColTypes().stream().map(
                         e -> e.toSql()).collect(Collectors.toList());
                 info.add(Joiner.on(", ").join(cols));
-                info.add(String.valueOf(!unstableGroups.contains(groupId)));
+                info.add(String.valueOf(!isGroupUnstable(groupId)));
```

Comment:
may need mark a TODO to cache the unstable result? queryStable API call may be costly.

Reply:
in FE `stable` is persisted, i think starmgr should also do this

Reply:
FE doesn't have multiple worker groups feature, the situation is different. Will consider how to cache the result if possible.

Reply:
better make application easier